docs/core/extensions/httpclient-http3.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Use HTTP/3 with HttpClient
+description: Learn how to use the HttpClient to access HTTP/3 servers in .NET 6
+author: IEvangelist
+ms.author: samsp
+ms.date: 09/02/2021
+---
+
+# Use HTTP/3 with HttpClient
+
+[HTTP/3](https://quicwg.org/base-drafts/draft-ietf-quic-http.html) is the third and upcoming major version of HTTP. HTTP/3 uses the same semantics as HTTP/1.1 and HTTP/2: the same request methods, status codes, and message fields apply to all versions. The differences are in the underlying transport. Both HTTP/1.1 and HTTP/2 use TCP as their transport. HTTP/3 uses a new transport technology developed alongside HTTP/3 called [QUIC](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport-34).
+
+HTTP/3 and QUIC have a number of benefits compared to HTTP/1.1 and HTTP/2:
+
+- Faster response time of the first request. QUIC and HTTP/3 negotiates the connection in fewer round-trips between the client and the server. The first request reaches the server faster.
+- Improved experience when there is connection packet loss. HTTP/2 multiplexes multiple requests via one TCP connection. Packet loss on the connection affects all requests. This problem is called "head-of-line blocking". Because QUIC provides native multiplexing, lost packets only impact the requests where data has been lost.
+- Supports transitioning between networks. This feature is useful for mobile devices where it is common to switch between WIFI and cellular networks as a mobile device changes location. Currently HTTP/1.1 and HTTP/2 connections fail with an error when switching networks. An app or web browsers must retry any failed HTTP requests. HTTP/3 allows the app or web browser to seamlessly continue when a network changes. HttpClient and Kestrel do not support network transitions in .NET 6. It may be available in a future release.
+
+> **Important**
```

Comment:
We use the MS.DOCS styling system for these. These are called ALERTS in the docs plugin for VSCode (if you want to find them and see the ones to choose from.
```suggestion
> [!IMPORTANT]
```